### PR TITLE
[MINOR] refactor: Reduce minor logs

### PR DESCRIPTION
**Pull request by wForget**
### What changes were proposed in this pull request?

reduce minor logs

### Why are the changes needed?

a lot of minor rss logs are printed in spark executor.

image

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

minor fix

### CODE REVIEW

**Comment by zuston** [comment]
Format code: `mvn spotless:apply -Pspark3 -Pspark2 -Ptez -Pmr -Phadoop2.8`

**Comment by wForget** [comment]
> Format code: `mvn spotless:apply -Pspark3 -Pspark2 -Ptez -Pmr -Phadoop2.8`

Thanks, fixed.

**Review by zuston — APPROVED** [review]
lgtm. Thanks @wForget

File: client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java [hunk 1]
```diff
@@ -64,9 +64,9 @@ public BufferManagerOptions(SparkConf sparkConf) {
             RssSparkConfig.RSS_WRITER_PRE_ALLOCATED_BUFFER_SIZE.defaultValue().get());
     requireMemoryInterval = sparkConf.get(RssSparkConfig.RSS_WRITER_REQUIRE_MEMORY_INTERVAL);
     requireMemoryRetryMax = sparkConf.get(RssSparkConfig.RSS_WRITER_REQUIRE_MEMORY_RETRY_MAX);
-    LOG.info(RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key() + "=" + bufferSize);
-    LOG.info(RssSparkConfig.RSS_WRITER_BUFFER_SPILL_SIZE.key() + "=" + bufferSpillThreshold);
-    LOG.info(
+    LOG.debug(RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key() + "=" + bufferSize);
```
Comment (rickyma) [inline client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java:67]: We should use `if (LOG.isDebugEnabled())` ahead of `LOG.debug`.

Comment (wForget) [reply client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java:67]: Thank you for your reminder, I will send a followup pr later.

Comment (zuston) [reply client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java:67]: > We should use `if (LOG.isDebugEnabled())` ahead of `LOG.debug`.

If don't have the time consuming var, the log.debug without `LOG.isDebugEnabled()` the is OK.

Comment (EnricoMi) [reply client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java:67]: The `LOG.isDebugEnabled())` is not needed if you would use string interpolation:
```suggestion
    LOG.debug("{}={}", RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key(), bufferSize);
```

Comment (rickyma) [reply client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java:67]: To make it simpler, we can always use `LOG.isDebugEnabled())` to guard `LOG.DEBUG`.

**Pull request merged**